lisp-modules: replace with an implementation based on github:uthar/nix-cl (#218870)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2305.section.md

- `boot.initrd.luks.device.<name>` has a new `tryEmptyPassphrase` option, this is useful for OEM's who need to install an encrypted disk with a future settable passphrase

> - Lisp gained a [manual section](https://nixos.org/manual/nixpkgs/stable/#lisp), documenting a new and backwards incompatible interface. The previous interface will not be removed until a future release.

Reviewer: I would word assertively…

```suggestion
- Lisp gained a [manual section](https://nixos.org/manual/nixpkgs/stable/#lisp), documenting a new and backwards incompatible interface. The previous interface will be removed in a future release.
```
